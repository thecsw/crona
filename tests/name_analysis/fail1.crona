a : int;

main : void(b : string) {
     a = b;
}