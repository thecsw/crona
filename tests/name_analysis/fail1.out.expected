a(int) : int;
main(byte array [0]->void) : void(b(byte array [0]) : byte array[0]){
	a(int) = b(byte array [0]);
}